org.wezfurlong.wezterm.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "org.wezfurlong.wezterm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "wezterm",
+    "finish-args": [
+        "--share=ipc",
+        "--filesystem=home:ro",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.portal.Desktop",
+        "--talk-name=org.freedesktop.portal.Settings",
+        "--share=network"
```

Comment:
I assume the main use-case is to launch a shell on the host. Does it need explicit network access in that case?

Reply:
network is used for wezterm to check the current release and inform the user about it. https://wezfurlong.org/wezterm/config/lua/config/check_for_updates.html has more information.